### PR TITLE
Allow server to force-reload client browsers

### PR DESCRIPTION
**Pull request by raphaelm**
Test command:

```
docker-compose exec server python manage.py connections force_reload "*"
```

### CODE REVIEW

File: webapp/src/lib/WebSocketClient.js [hunk 1]
```diff
@@ -188,6 +190,11 @@ class WebSocketClient extends EventEmitter {
 		}
 	}
 
+	_handleReload (message) {
+		this._reconnect = false;
```
Comment (rashfael) [inline webapp/src/lib/WebSocketClient.js:194]: I'd rather just normally close the connection instead of an extra flag.

Comment (raphaelm) [reply webapp/src/lib/WebSocketClient.js:194]: makes sense, I wasn't sure if that would trigger the reconnect again

Comment (raphaelm) [reply webapp/src/lib/WebSocketClient.js:194]: … or cause a reconnect if the server is closing the connection quicker than we are ;) but I can stop the server from closing after this message as ewll

Comment (rashfael) [reply webapp/src/lib/WebSocketClient.js:194]: If the server closes the connection before the message is handled it's too late anyway. Not sure if the browser does those in order.

**Pull request merged**